index.mjs

```diff
@@ -0,0 +1,7 @@
+import * as browserSDK from './build/browser/index.mjs';
+import * as nodeSDK from './build/node/index.mjs';
```

Comment:
is this going to include both builds in downstream bundles?

Reply:
And also this would just blow away when we decide to convert this to a mono-repo and create 1 package for each platform :)